resources/js/components/ReadMoreButton.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-    <span class="underline text-gray-400 cursor-pointer text-xs" @click="sendReadmoreEvent">Read more</span>
+    <span dusk="btn__readmore" class="underline text-gray-400 cursor-pointer text-xs" @click="sendReadmoreEvent">
```

Comment:
this needs to be unique per element, i saw some page has more than 1 read more link